output/rewrite_tag_filter.md

```diff
@@ -36,6 +38,59 @@ Sample data:
 +------------------------------------------+        +------------------------------------------------+
 ```
 
+#### Nested kubernetes namespace attributes based rules
+
+This is an example of how to use this plugin to rewrite tags with nested attributes which are kubernetes metadata. In the example, records tagged with `kubernetes.information` will have their tag prefixed with the value of the nested key `kubernetes.namespace_name`.
+
+##### Dot notation
+
+```text
+<match kubernetes.**>
+  @type rewrite_tag_filter
+  <rule>
+    key $.kubernetes.namespace_name
+    pattern ^(.+)$
+    tag $1.${tag}
+  </rule>
+</match>
+```
+
+##### Bracket notation
+
+```text
+<match kubernetes.**>
+  @type rewrite_tag_filter
+  <rule>
+    key $['kubernetes']['namespace_name']
+    pattern ^(.+)$
+    tag $1.${tag}
+  </rule>
+</match>
+```
+
+Sample data:
+
+```text
++----------------------------------------------------------------------------------------+        +-------------------------------------------------------------------------------------------------------+
+| original record                                                                        |        | rewritten tag record                                                                                  |
+|----------------------------------------------------------------------------------------|        |-------------------------------------------------------------------------------------------------------|
+| kubernetes.information {"kubernetes" : { "namespace_name": "kube-system" }, ... }      | +----> | kube-system.kubernetes.information {"kubernetes" : { "namespace_name": "kube-system" }, ... }         |
+| kubernetes.information {"kubernetes" : { "namespace_name": "default" }, ... }          | +----> | default.kubernetes.information {"kubernetes" : { "namespace_name": "default" }, ... }                 |
+| kubernetes.information {"kubernetes" : { "namespace_name": "kube-public" }, ... }      | +----> | kube-public.kubernetes.information {"kubernetes" : { "namespace_name": "kube-public" }, ... }         |
+| kubernetes.information {"kubernetes" : { "namespace_name": "kube-node-lease" }, ... }  | +----> | kube-node-lease.kubernetes.information {"kubernetes" : { "namespace_name": "kube-node-lease" }, ... } |
++----------------------------------------------------------------------------------------+        +-------------------------------------------------------------------------------------------------------+
+```
+
+These example copnfigurations can process nested attributes like following:
+
+```json
+{
+  "kubernetes": {
+    "namespace_name": "kube-system"
+  }
+}
+```
+
```

Comment:
It may be redundant, but not sure. (because the original record is shown in table)

Reply:
Hmm....
I decided to remove from the article.
This seems to be redundant. :(